<ui version="4.0" >
 <author>Dane Springmeyer</author>
 <class>Fishery</class>
 <widget class="QDialog" name="Fishery" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Interview</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>63</height>
      </size>
     </property>
     <property name="title" >
      <string>Fishery Sector</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="fishery_sector_label" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Dynamic Fishery Sector Name</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="vessel_info" >
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title" >
        <string>Fisher Information</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" native="1" name="vessel_length" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="vessel_length_text" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Vessel length (meters):</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="vessel_length_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="contextMenuPolicy" >
              <enum>Qt::PreventContextMenu</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="vessel_length_2" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="vessel_motor_text" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Vessel motor (horsepower):</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="vessel_motor_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="vessel_length_3" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="haul_capacity_text" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Haul Capacity (kilograms):</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="haul_capacity_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="home_port_text" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Home Port:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="home_port_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port_4" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="landing_port_text" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Landing Port 1:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="landing_port_line" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port_7" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="landing_port_text_2" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Landing Port 2:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="landing_port_line_2" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port_8" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="landing_port_text_3" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Landing Port 3:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="landing_port_line_3" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="home_port_9" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="landing_port_text_4" >
             <property name="minimumSize" >
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string>Landing Port 4:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="landing_port_line_4" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="minimumSize" >
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font" >
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="button_box" >
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbnSelectGear" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Draw Shapes by Gear Type</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
